index.js

```diff
@@ -11,9 +11,13 @@ function menuClickedHandler() {
 
 document.getElementById('menu').addEventListener('click', menuClickedHandler);
 
-function btn() {
-  const cardone = document.getElementById('cardwork1');
-  cardone.classList.toggle('popup-windows');
-}
-
-document.getElementById('btn1').addEventListener('click', btn);
+document.getElementById('contact-form').addEventListener('submit', (e) => {
+  const errorMsg = document.getElementById('error-message');
```

Comment:
- [ ] Kindly fix the error, whenever i submit the email field with uppercase i don't get an error message. I even clone the project locally and run same error.

Reply:
Thank you so much for finding out my mistakes